src/ui/loading-view.js

```diff
@@ -32,8 +32,13 @@ export class LoadingView {
     /** @private @const {!Document} */
     this.doc_ = doc;
 
+    this.loadingContainer_ = createElement(this.doc_, 'div', {
+      'class': 'swg-loading-container',
```

Comment:
Use `swg-loading-container` as the tag name per previous change. Also make sure you immediately add `swg-loading-container {display: block}`.